### PR TITLE
Add schema for creating movie and source tables

### LINKED ISSUE #37: [3pt] Create a schema.sql file with tables that will be analogical to the structure of the classes that you created (TM 4)
- movies table (add all properties and associations from the parent Item class as table columns)
- sources table

### PR DESCRIPTION
**Pull request by H-Ramalan**
- Add schema for creating movie and source tables

### CODE REVIEW

File: sql/schema.sql [hunk 1]
```diff
@@ -61,3 +61,19 @@ CREATE TABLE label (
...
   color varchar(100)
 );
+
+-- Create Movie Table
+CREATE TABLE movie (
+  id  integer GENERATED ALWAYS AS IDENTITY,
+  publish_date  date,
+  silent boolean,
+  source_id  int references source(id),
+  PRIMARY KEY(id)
```
Comment (rubydevi) [inline sql/schema.sql:71]: The movie table looks good, you have included all the necessary columns. You can add one more column to connect the movie table and the item table like an item_id fk in the movie table and you are done 👍

**Review by rubydevi — CHANGES_REQUESTED** [review]
# Hi @H-Ramalan 

You have done a great job!!!

## Highlights
- Added schema for creating movie and source tables ✔
- All the tables are working fine ✔
- Correct use of Git Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- Please check the comment below to make the required changes 👍 




Cheers and Happy coding!👏👏👏

File: sql/schema.sql (hunk 1 above)
Comment (H-Ramalan) [reply sql/schema.sql:72]: Made requested changes to the movie table. Thank you for the correction @rubydevi

Comment (rubydevi) [reply sql/schema.sql:72]: Great job implementing the required change. 👍

**Review by rubydevi — APPROVED** [review]
Hi @H-Ramalan ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

**Pull request merged**